core/templates/pages/contributor-dashboard-page/contributions-and-review/contributions-and-review.component.ts

```diff
@@ -408,8 +410,15 @@ export class ContributionsAndReview
       this.contextService.setCustomEntityContext(
         AppConstants.IMAGE_CONTEXT.EXPLORATION_SUGGESTIONS,
         suggestion.target_id);
+      const suggestionIdsList = Object.keys(this.contributions);
+      const clickedIndex = suggestionIdsList.indexOf(suggestionId);
+      const skippedContributionIds = [];
+      for (let i = 0; i < clickedIndex; i++) {
+        skippedContributionIds.push(suggestionIdsList[i]);
+      }
```

Comment:
We can use slice here:

```suggestion
      const suggestionIds = Object.keys(this.contributions);
      const clickedSuggestionIndex = suggestionIdsList.indexOf(suggestionId);
      const skippedContributionIds = suggestionIds.slice(0, clickedIndex);
```

Let's also move this logic to core/templates/pages/contributor-dashboard-page/modal-templates/translation-suggestion-review-modal.component.ts, since that component is already computing "active suggestion" state.

Reply:
Done.